### DISCUSSION TITLE
Maybe a NEWB question...but how do I get this to pull templates into my netbox?

### DISCUSSION THREAD

**Discussion by TheDirkMelvin** [netbox-community/devicetype-library#634] [Q&A] [answered]
I am hoping this is a silly thing that I am missing, but I don't see instructions on how to get the device templates from this git, into my netbox installation.

**Comment by ryanmerolle** [comment, marked as answer]
I think we need to do a better job of documenting.  It may be in the read the docs for NetBox, but I could not find it.

We should probably add to the wiki for NetBox, the Read the Docs, and the README for this project.

Example url to add the yamls: https://demo.netbox.dev/dcim/device-types/import/

Under the Devices section there is Device-Types.  Just click the import button and it will bring you to the above link.  You can then paste a the yaml file from this repo.  There are some unoffical projects to auto import from this project, but given that its pretty common to have a manageable number of model variations, individual importing is usually not a problem.  It also allows you to tweak the model before importing because you want to document that you only deploy this model with a particular card always installed or a particular PSU that is not standard.

**Reply by ryanmerolle** [reply]
Feel free to contribute a blurb in the readme to give back :)
